database/3ds/get_friend_miis.go

```diff
@@ -4,13 +4,13 @@ import (
 	"database/sql"
 	"time"
 
-	"github.com/PretendoNetwork/friends-secure/database"
-	"github.com/PretendoNetwork/friends-secure/globals"
+	"github.com/PretendoNetwork/friends/database"
+	"github.com/PretendoNetwork/friends/globals"
 	"github.com/PretendoNetwork/nex-go"
 	friends_3ds_types "github.com/PretendoNetwork/nex-protocols-go/friends-3ds/types"
 )
 
-// Get a friend's mii
+// GetFriendMiis returns the Mii of all friends
 func GetFriendMiis(pids []uint32) []*friends_3ds_types.FriendMii {
```

Comment:
Since we're here, this should also return an error. The logging and whether or not the request should respond with an error should be done in the request handler honestly, putting the logging in this functions doesn't make a _ton_ of sense